src/Analyser/NodeScopeResolver.php

```diff
@@ -371,9 +371,9 @@ private function processNode(\PhpParser\Node $node, Scope $scope, \Closure $node
 		} elseif (
 			$node instanceof \PhpParser\Node\Expr\StaticCall
 			&& $node->class instanceof \PhpParser\Node\Name
-			&& is_string($node->name)
+			&& is_string($node->name->name)
```

Comment:
`StaticCall::$name` is either: `string|Identifier|Expr` so it's not safe to access `$node->name->name` immediately...